### PR TITLE
SVG

### PR DESCRIPTION
**Pull request by abp437**
This PR Intends to clear up the use cases of using SVGs and avoid usage of Sprite and replace it with SVGs.

### CODE REVIEW

**Review by hitesh-patil — APPROVED** [review]

File: SVG/Optimised/css/svg.css [hunk 1]
```diff
@@ -0,0 +1,5 @@
+.featuredIcons svg {
+	height: 60px;
+	margin: 0 auto;
```
Comment (shaileshpatil-work) [inline SVG/Optimised/css/svg.css:3]: No need to use

File: SVG/SVG Use Cases/css/svgUseCases.css [hunk 2]
```diff
@@ -0,0 +1,101 @@
+.featuredIcons svg {
+	min-height: 60px;
+	margin: auto;
+}
+
+
+.priceContainer img {
+	display: block;
```
Comment (shaileshpatil-work) [inline SVG/SVG Use Cases/css/svgUseCases.css:8]: Indentation missing

File: SVG/SVG Use Cases/css/svgUseCases.css [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+  height: 44px;
+  min-height: 60px;
+  margin: auto;
+}
+
+.passengerIcon {
+	display: block;
+  width: 55px;
+  height: 60px;
+  background: url('../svg/passengerIcon.svg');
```
Comment (shaileshpatil-work) [inline SVG/SVG Use Cases/css/svgUseCases.css:19]: Use background shorthand property

File: SVG/SVG Use Cases/css/svgUseCases.css [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+.priceContainer img {
+	display: block;
+  width: 27px;
+  height: 44px;
+  min-height: 60px;
+  margin: auto;
+}
+
+.passengerIcon {
+	display: block;
```
Comment (shaileshpatil-work) [inline SVG/SVG Use Cases/css/svgUseCases.css:16]: Indentation missing

File: SVG/SVG Use Cases/css/svgUseCases.css [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+  width: 55px;
+  height: 60px;
+  background: url('../svg/passengerIcon.svg');
+  background-size: 55px 60px;
+	background-repeat: no-repeat;
+	margin: 0 auto;
+}
+
+.flightIconContainer {
+	display: block;
```
Comment (shaileshpatil-work) [inline SVG/SVG Use Cases/css/svgUseCases.css:26]: Indentation missing

File: SVG/Optimised/optimisedSvg.html [hunk 6]
```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no">
```
Comment (shaileshpatil-work) [inline SVG/Optimised/optimisedSvg.html:5]: Check Indaentaion

File: SVG/SVG Use Cases/css/svgUseCases.css [hunk 7]
```diff
@@ -0,0 +1,101 @@
...
+  margin: auto;
+}
+
+.passengerIcon {
+	display: block;
+  width: 55px;
+  height: 60px;
+  background: url('../svg/passengerIcon.svg');
+  background-size: 55px 60px;
+	background-repeat: no-repeat;
```
Comment (shaileshpatil-work) [inline SVG/SVG Use Cases/css/svgUseCases.css:21]: Indentation missinng

File: SVG/SVG Use Cases/css/svgUseCases.css [hunk 8]
```diff
@@ -0,0 +1,101 @@
...
+}
+
+.flightIconContainer {
+	display: block;
+  width: 52px;
+  height: 61px;
+	margin: 0 auto;
+}
+
+.passengerIcon {
```
Comment (shaileshpatil-work) [inline SVG/SVG Use Cases/css/svgUseCases.css:32]: Repeatedly use same css check line no 15

File: SVG/SVG Use Cases/css/svgUseCases.css (hunk 2 above)
Comment (shaileshpatil-work) [inline SVG/SVG Use Cases/css/svgUseCases.css:8]: Write display:inline-block and remove margin:auto as wrapper have text-align:center

**Review by shaileshpatil-work — COMMENTED** [review]
Check

**Pull request merged**